content/blog/intro-to-html-css-and-savascript/index.md

```diff
@@ -0,0 +1,71 @@
+
+HTML also serves as the foundation for web accessibility for people with disabilities. [Screen readers](https://www.afb.org/blindness-and-low-vision/using-technology/assistive-technology-products/screen-readers) allow people who are blind or have low vision to use the internet in a primarily audio format. They work by reading the HTML directly and are programmed to follow its structure and pick up on information included in the element tags. Practicing good [web accessibility standards](https://www.w3.org/WAI/standards-guidelines/wcag/) is important for a modern web developer and makes websites that are easier to use for everyone.
+
+
+## CSS
+
+CSS, or [Cascading Style Sheets](https://developer.mozilla.org/en-US/docs/Web/CSS), is what web developers use to control the look and feel of the websites they create.
+
+Let’s think back to our building construction analogy for a moment. We have our steel frame (HTML) but what will the building look like when it is finished? Will the facade be brick, wood siding, or cut stone? Should it be painted? If so, what color? These are the kinds of questions that web developers use CSS to answer. 
+
+CSS can be used to control virtually every visual aspect of a website. Take a look at the site you are reading right now. Everything on it is controlled (or “styled” as the lingo goes) by CSS. The background color, the color of the text, the size of the text, where images or videos appear on the page, the location of the navigation bar, and even some animations are set by CSS. 
+
+In modern web development, CSS is written in a separate document  called a “stylesheet” that is referenced in the HTML document with a link element. The stylesheet contains a list of HTML elements, identified by various “selectors”, that the developer wants to style. Each element may contain any number of [properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Properties_Reference) and property values that set up how that element will be presented to the user in the web browser. These properties have a specific mapping to various aspects of how the page is styled. For example, `font-size` dictates the size of the font of the text on the page.  Exactly what appears on the screen can be a little tricky for a beginner and I encourage you to read the excellent [tutorial in Mozilla’s documentation](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Cascade_and_inheritance). For now, just know that CSS rules can inherit property values from each other and can even override each other in certain circumstances. 
+
+CSS is a big topic and you will very likely be learning more about it as long as you do web development. There are pre-made frameworks, preprocessors, and lots of other CSS-related goodies out there to dive into as you learn your new craft. The important thing at the outset is to learn the basics well so you know what to expect as you write your own CSS stylesheets.
```

Comment:
```suggestion
CSS is a big topic, and you will very likely be learning more about it as long as you do web development. There are pre-made frameworks, preprocessors, and lots of other CSS-related goodies out there to dive into as you learn your new craft. The important thing at the outset is to learn the basics well so that you know what to expect as you write your own CSS stylesheets.
```